### PR TITLE
Homepage and routing

### PR DESCRIPTION
**Pull request by brybusti**

### CODE REVIEW

**Comment by Rhaypapenfuzz** [comment]
I think app.js and app.css are conflicting

**Comment by Rhaypapenfuzz** [comment]
make a copy of your work and try pulling from master as well. In case you lose any info, you can easily restore it

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,148 @@
...
+      //     <a
+      //       className="App-link"
+      //       href="https://reactjs.org"
+      //       target="_blank"
+      //       rel="noopener noreferrer"
+      //     >
+      //       Learn React
+      //     </a>
+      //   </header>
+      // </div>
```
Comment (Azeezah) [inline src/App.js:123]: There's some commented out code here.

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,148 @@
...
+    /*
+    var provider = new firebase.auth.GoogleAuthProvider(); 
+    firebase.auth().signInWithPopup(provider).then(function(result) {
+      console.log(result.user);
+      setUser(result.user);
+    }).catch(function(error) {
+      console.log(error);
+    });
+    */
+  } // End of loginPress
```
Comment (Azeezah) [inline src/App.js:52]: Let's remove these comments as well.

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,148 @@
...
+  Link
+} from "react-router-dom";
 import './App.css';
+import lock from './lock.png'
+import unlock from './unlock.png'
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Authentication from './services/authentication.js';
+//import ButtonAppBar from './nav.js';
+//import TemporaryDrawer from './navSide.js';
```
Comment (Azeezah) [inline src/App.js:15]: Ditto.

File: src/App.css [hunk 4]
```diff
@@ -7,18 +7,23 @@
   pointer-events: none;
 }
 
-@media (prefers-reduced-motion: no-preference) {
+/* @media (prefers-reduced-motion: no-preference) {
   .App-logo {
     animation: App-logo-spin infinite 20s linear;
   }
-}
+} */
```
Comment (Azeezah) [inline src/App.css:14]: Ditto.

**Review by Azeezah — CHANGES_REQUESTED** [review]

**Pull request closed without merging**